bedrock/firefox/templates/firefox/index.html

```diff
@@ -75,21 +75,21 @@ <h1 class="mzp-has-zap-11">
           'height': '233'
         }
       ) }}
-      <h2 class="c-landing-grid-title"><a href="{{ url('firefox.new') }}" data-cta-type="link" data-cta-text="Desktop">{{ ftl('firefox-browsers-desktop') }}</a></h2>
+      <h2 class="c-landing-grid-title"><a href="{{ url('firefox.new') }}" data-cta-text="Desktop">{{ ftl('firefox-browsers-desktop') }}</a></h2>
       <p>{{ ftl('firefox-browsers-seriously-private-browsing') }}</p>
 
       <p id="desktop-download">
         {# Old IE users need to click a download button, the JS on the thank you page doesn't get them the right download if we send them there directly #}
         <!--[if IE]>
-          <a href="{{ url('firefox.new') }}" data-cta-type="link" data-cta-text="Firefox Desktop">{{ ftl('firefox-browsers-download-for-desktop') }}</a>
+          <a href="{{ url('firefox.new') }}" data-cta-text="Firefox Desktop">{{ ftl('firefox-browsers-download-for-desktop') }}</a>
         <![endif]-->
         <!--[if !IE]><!-->
           {# Download link should be locale neutral see issue 7982 #}
-          <a id="qa-desktop-download" class="mzp-c-cta-link cta-download" href="/firefox/download/thanks/" data-link-type="Desktop" data-download-os="Desktop" data-download-version="standard" data-testid="firefox-desktop-download">{{ ftl('firefox-browsers-download-for-desktop') }}</a>
+          <a id="qa-desktop-download" class="mzp-c-cta-link cta-download" href="/firefox/download/thanks/" data-link-type="download" data-download-os="Desktop" data-download-version="standard" data-testid="firefox-desktop-download">{{ ftl('firefox-browsers-download-for-desktop') }}</a>
```

Comment:
Note: This is a correction of a code error in a data-link-type="download" attribute. I'm going to have to deal with these links in a separate PR #14062